sigs.yaml

```
@@ -0,0 +1,25 @@
+sigs:
+- dir: sig-operations
+  name: Operations
+  mission_statement: >
+    The goal of this SIG is to enable management and operations for hardware and platform tooling within the Operate
```

It seems to me that `enable` is doing a LOT of work as a verb here, and it's not a very clear one in this case. It feels like we're looking e.g.:

```suggestion
    The goal of this SIG is to define and perform management and operations for hardware and platform tooling within the Operate
```

---

Our way of thinking was, that the SIG does not perform stuff, but subprojects do. Might be nit-picking.
How about

```suggestion
    The goal of this SIG is to define and coordinate management and operations for hardware and platform tooling within the Operate
```

---

Updated